.prettierrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   tabWidth: 2,
   trailingComma: 'es5',
   useTabs: false,
+  ignore: ['**/*.svg'],
   overrides: [
```

Comment:
```suggestion
  overrides: [
    {
      files: ['**/*.svg'],
      options: { parser: 'html' },
    },
```

Short of using the [prettier XML plugin](https://github.com/prettier/plugin-xml), we should parse SVG as HTML. We can review this later on, but this should be fine for now.

Refs: https://github.com/prettier/prettier/issues/5322#issuecomment-1450182797